python/ray/ml/examples/tf/tune_tensorflow_mnist_example.py

```diff
@@ -4,7 +4,7 @@
 from ray import tune
 from ray.ml.train.integrations.tensorflow import TensorflowTrainer
 
-from ray.ml.examples.tensorflow.tensorflow_mnist_example import train_func
+from ray.ml.examples.tf.tensorflow_mnist_example import train_func
```

Comment:
reason for this rename is that otherwise the examples won't work if executed from the examples working dir (as we try to `import tensorflow as tf` - and it will use the local directory then)